types/types.go

```diff
@@ -466,6 +466,18 @@ type ImageInspectInfo struct {
 	Os            string
 	Layers        []string
 	Env           []string
+	LayersDetail  []LayerDetail
+	History       []v1.History
+	Author        string
+	Size          int64
+	Config        v1.ImageConfig
```

Comment:
Adding a full config here is redundant with `Image.OCIConfig`, and probably unwanted for the `Inspect` caller, who is calling `Inspect` exactly not to deal with the details of the config format.

Reply:
I use podman  or docker inspect an image, they will show config, so I think we should stay the same.

Reply:
But then are other callers like `skopeo inspect`, or automation that just wants a single element (like looking up a single label) that don’t need both at the same time. Paying the cost of the full conversion and JSON formatting is not all that great for them.

As a maintainer, I’d also _much_ prefer not to have to maintain logic to convert to OCI format in two separate implementations for each format.